sql/src/main/java/io/crate/planner/node/dql/MergePhase.java

```diff
@@ -137,6 +138,49 @@ public static MergePhase sortedMerge(UUID jobId,
         return mergeNode;
     }
 
+    @Nullable
+    public static MergePhase mergePhase(ConsumerContext context,
+                                        Collection<String> executionNodes,
+                                        UpstreamPhase upstreamPhase,
+                                        @Nullable OrderBy orderBy,
+                                        @Nullable List<? extends Symbol> orderBySymbols,
+                                        List<Projection> projections,
+                                        List<Symbol> inputs,
+                                        @Nullable List<DataType> inputTypes,
```

Comment:
Might make sense to describe some of the arguments? E.g. why does it make sense to declare inputTypes? Would seem that they can always be retrieved from the inputs.